akka-javasdk/src/main/java/akka/javasdk/agent/CoreMemory.java

```diff
@@ -8,47 +8,31 @@
 
 package akka.javasdk.agent;
 
-import java.util.List;
-
 /**
  * Interface for managing conversation history between users and AI models, using CoreMemory
  * provides functionality to store, retrieve, and manage messages exchanged during conversations in
  * an agent system.
  */
 public interface CoreMemory {
```

Comment:
I'm not sure about the name `CoreMemory.` How about `AgentMemory`?

Reply:
Maybe we can have this discussion on the side?

Reply:
in any case, I wont change those names in this PR.